src/sign/index.ts

```diff
@@ -107,6 +107,13 @@ export default class Client {
       fetch(address, init)
         .then((res) => {
           if (res.ok) return resolve(res.json());
+          if (res.headers.get('content-type')?.includes('application/json'))
+            return res
+              .json()
+              .then((e) => reject(e))
```

Comment:
SHould not reject here

Reply:
If we don't reject, UI can't catch it https://github.com/snapshot-labs/snapshot/blob/master/src/composables/useClient.ts#L28-L36